### DISCUSSION TITLE
Links in the production-site pointing to my staging repo

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1793] [Q&A]
@MoBosse @brockfanning 

Yesterday i created a productiuon site for my platform: [https://mt-production.github.io/](https://mt-production.github.io/)

Everthing works well. On my Migration report I have an excel file for each indicator, which contains the data formatted as the publication policy of my office stated. These excel files can be found in the meta data tab "Tabelle zum Download" 

e.g.  the first indicator: [https://mt-production.github.io/1-1-1/](https://mt-production.github.io/1-1-1/)
the link to the excel file points to my staging repo .... :[https://christoph-lsn.github.io/MT_Site/assets/excel/2022_1-1-1.xlsx](https://christoph-lsn.github.io/MT_Site/assets/excel/2022_1-1-1.xlsx)

It works, but it would be great if it ponts to the production site: https://github.com/MT-production/MT-production.github.io/tree/master/assets/excel

I thouht about absolut and relative links but I don't know how to use relative links in my meta *.md files in open-sdg.....

the md file for indicator 1.1.1: https://github.com/christoph-LSN/IM_Data/blob/develop/meta/1-1-1.md
```
source_active_1: true
source_url_1: 'https://christoph-lsn.github.io/MT_Site/assets/excel/2022_1-1-1.xlsx'
source_url_text_1: global_indicators.1-1-1-title
```

**Comment by MoBosse** [comment]
@christoph-LSN Not sure about that since I link directly to the production environment in those cases but you could try this:
`source_url_1: '{{ site.baseurl }}/assets/excel/2022_1-1-1.xlsx'`

**Reply by brockfanning** [reply]
@MoBosse @christoph-LSN Linking directly to the file on the production environment is certainly a good and simple solution.

I don't think using `{{ site.baseurl }}` would work - I believe those Jekyll/Liquid templating variables only work when they are in files in the _includes, _layouts, _pages, and _posts folders. In other words they work when they are in standard Jekyll stuff, but not necessarily when they are in custom Open SDG stuff - like metadata content. I think we would would need to add special code to Open SDG to support the use of {{ site.baseurl }} inside of metadata content.

**Comment by brockfanning** [comment]
@christoph-LSN I think you asked the same question in #1520 - I added a reply over there with a potential fix.
